ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EjbJndiBindingsDeploymentUnitProcessor.java

```diff
@@ -128,7 +122,8 @@ private void setupJNDIBindings(EJBComponentDescription sessionBean, DeploymentUn
 
         // the base ServiceName which will be used to create the ServiceName(s) for each of the view bindings
         final StringBuilder jndiBindingsLogMessage = new StringBuilder();
-        jndiBindingsLogMessage.append("JNDI bindings for session bean named " + sessionBean.getEJBName() + " in deployment unit " + deploymentUnit + " are as follows:\n\n");
+        jndiBindingsLogMessage.append("JNDI bindings for session bean named " + sessionBean.getEJBName() + " in deployment unit " + deploymentUnit + " are as follows:");
```

Comment:
Don't remove it, as it's important logging. You don't even need to deal with it as part of this PR; I added the comment mostly as an FYI.